st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -78,16 +78,18 @@ def match(self, action_alias_match_api):
         command = action_alias_match_api.command
 
         try:
-            match = get_matching_alias(command=command)
+            format_ = get_matching_alias(command=command)
         except ActionAliasAmbiguityException as e:
             LOG.exception('Command "%s" matched (%s) patterns.', e.command, len(e.matches))
             return abort(http_client.BAD_REQUEST, str(e))
 
         # Convert ActionAliasDB to API
-        action_alias_api = ActionAliasAPI.from_model(match[0])
-        match = [action_alias_api, match[1], match[2]]
-        result = self._match_tuple_to_dict(match=match)
-        return result
+        action_alias_api = ActionAliasAPI.from_model(format_['alias'])
```

Comment:
Does this change the response format of match API endpoint?

Because if it does, it's not backward compatible (match and match and execute are two different endpoints, match and execute is the recently added one which didn't have any official consumers yet).

Reply:
Never mind, I see `_match_to_tuple` change below :D

Reply:
The variable named `match` wasn't very clear the type of object it was matching (alias/format/representation?), so I renamed it to `format_` to make it more clear (and to avoid stomping on the `format` built-in).